db/commands/station.js

```diff
@@ -0,0 +1,131 @@
+const db = require( '../config' ).db
+const functions = require( '../functions')
+
+class Station {
+
+  constructor( stationData ) {
+    this.stationName = stationData.stationName
+    this.stationNumber = stationData.stationNumber
+  }
+
+  static getStationID( stationName ) {
+    let getStationId =
+      `SELECT station_number
+        FROM stations
+          WHERE station_name = $1`
+    return db.one( getStationId, stationName )
+    .then( station => station.station_number )
+  }
+
+  static getStationLocation( stationNumber ) {
+    let getStation =
+      `SELECT * FROM stations
+        WHERE station_number = $1`
+    return db.one( getStation, stationNumber)
+    .then( station => station.station_name )
+  }
+
+  static getWaitingPassengers( stationName ) {
+    let getPassengersAtStation =
+      `SELECT * FROM passengers WHERE station_name = $1`
+    return db.any( getPassengersAtStation, stationName )
+    .then( waitingPassengers => waitingPassengers )
```

Comment:
why are you doing this? `.then( waitingPassengers => waitingPassengers )` I keep seeing folks do this but I don't understand why. This shouldn't be needed and shouldn't make any difference. Does it?